app/javascript/components/subject/MonitoringStatus.js

```diff
@@ -193,8 +212,20 @@ class MonitoringStatus extends React.Component {
     this.setState({
       message: 'jurisdiction from "' + this.props.jurisdictionPaths[this.state.originalJurisdictionId] + '" to "' + this.state.jurisdictionPath + '".',
       showJurisdictionModal: !current,
-      jurisdiction: current ? this.state.jurisdictionPath : this.props.jurisdictionPaths[this.state.originalJurisdictionId], // Reset select jurisdiction if cancel
+      jurisdictionPath: current ? this.props.jurisdictionPaths[this.state.originalJurisdictionId] : this.state.jurisdictionPath,
       apply_to_group: false,
+      reasoning: '',
```

Comment:
this is to reset the reasoning field in case a user starts changing one of these monitoring action fields and writes a reasoning and then decides to cancel